### PR TITLE
fixed broken links in readme

### PR DESCRIPTION
**Pull request by Jayita10**
Issue: #16166 

## What I did

In the README file, fixed the links which are broken.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,8 +143,8 @@ See [Addon / Framework Support Table](https://storybook.js.org/docs/react/api/fr
 
 | Addons                                                                           |                                                            |
 | -------------------------------------------------------------------------------- | ---------------------------------------------------------- |
-| [info](https://github.com/storybookjs/deprecated-addons/tree/main/addons/info)   | Annotate stories with extra component usage information    |
-| [notes](https://github.com/storybookjs/deprecated-addons/tree/main/addons/notes) | Annotate Storybook stories with notes                      |
+| info   | Annotate stories with extra component usage information    |
```
Comment (yannbf) [inline README.md:146]: Actually the link was broken however the repo was correct. It was just a branch renaming issue. Could you please replace it with the correct link? Thanks!

```suggestion
| [info](https://github.com/storybookjs/deprecated-addons/tree/master/addons/info)   | Annotate stories with extra component usage information    |
```

**Review by yannbf — CHANGES_REQUESTED** [review]
Hey @Jayita10 thanks a lot for your contribution! I think there was a miscommunication and actually the links to info/notes should be kept, just fixed. Could you please do the changes? Thank you!

File: README.md [hunk 2]
```diff
@@ -103,11 +103,11 @@ For additional help, join us in the [Storybook Discord](https://discord.gg/story
...
+| [Mithril](storybookjs/mithril)                                         | [v6.3.x](https://storybookjs.netlify.com/mithril-kitchen-sink/)                             | [![Mithril](https://img.shields.io/npm/dm/@storybook/mithril.svg)](app/mithril)                      |
+| [Marko](storybookjs/marko)                                             | [v6.3.x](https://storybookjs.netlify.com/marko-cli/)                                        | [![Marko](https://img.shields.io/npm/dm/@storybook/marko.svg)](app/marko)                            |
+| [Riot](storybookjs/riot)                                               | [v6.3.x](https://storybookjs.netlify.com/riot-kitchen-sink/)                                | [![Riot](https://img.shields.io/npm/dm/@storybook/riot.svg)](app/riot)                               |
+| [Rax](storybookjs/rax)                                                 | [v6.3.x](https://storybookjs.netlify.com/rax-kitchen-sink/)                                 | [![Rax](https://img.shields.io/npm/dm/@storybook/rax.svg)](app/rax)                                  |
```
Comment (jonniebigodes) [inline README.md:110]: @Jayita10 if you wouldn't mind changing this to the following:
- Marionette.js should link to https://github.com/storybookjs/marionette
- Mithril should link to https://github.com/storybookjs/mithril
- Marko should link to https://github.com/storybookjs/marko
- Riot should link to https://github.com/storybookjs/riot
- Rax should link to https://github.com/storybookjs/rax


To give you a bit of context on this, up until recently, those frameworks were implemented within the app directory in the Storybook monorepo. With a recent change they were moved into their own separate repositories to allow any contributors to continue working on supporting them.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@Jayita10 I've checked and left one item from my end that needs addressing. Let me know once you've updated your pull request based on my and @yannbf 's feedback so that I can take another look at it and hopefully merge it.

Have a great day!

Stay safe

**Review by yannbf — APPROVED** [review]
Looks great to me! Thanks a lot for your contribution! ♥️

**Pull request merged**